klog.go

```diff
@@ -872,6 +872,16 @@ func kvListFormat(b *bytes.Buffer, keysAndValues ...interface{}) {
 	}
 }
 
+func StringertoString(s fmt.Stringer) (ret string) {
```

Comment:
nil: should that be `StringerToString`?

Reply:
More important: should that function be exported and thus be part of the klog API?

I prefer to keep it internal.

Reply:
will switch it to `stringertoString`